src/main/java/com/dnsimple/Registrar.java

```diff
@@ -160,6 +161,19 @@
    */
   public DisableWhoisPrivacyResponse disableWhoisPrivacy(String accountId, String domainId) throws DnsimpleException, IOException;
 
+  /**
+   * Renew whois privacy for the domain.
+   *
+   * @see <a href="https://developer.dnsimple.com/v2/registrar/whois-privacy/#disable">https://developer.dnsimple.com/v2/registrar/whois-privacy/#renew</a>
```

Comment:
You are linking to `#disable`

Reply:
Fixed by https://github.com/dnsimple/dnsimple-java/pull/12/commits/0630fa9216dc577109856804afca27afbe944c8e